docking/rigorous_work.py

```diff
@@ -21,28 +21,34 @@
 DELETION_MAX_LAMBDA = 1.0
 MIN_LAMBDA = 0.0
 INSERTION_STEPS = 501
-DELETION_STEPS = 501
 EQ1_STEPS = 5001
```

Comment:
Would it be helpful to expose any other of these variables as CLI arguments? (Maybe `INSERTION_STEPS` and `EQ1_STEPS`?)

Reply:
might as well!